docker/Dockerfile

```diff
@@ -117,7 +117,10 @@ RUN pip install --no-cache-dir nbresuse \
   && jupyter serverextension enable --py --system nbresuse \
   && pip install --no-cache-dir \
   --extra-index-url="https://packages.dea.ga.gov.au" \
-  --no-deps --upgrade odc-algo odc-ui 'datacube[performance]'
+  --no-deps --upgrade odc-algo odc-ui 'datacube[performance]' \ 
+  && mkdir -m664 /etc/jupyter
```

Comment:
pretty sure docker can manage creating subdirs, no need to add mkdir, you can also use `--owner` argument to COPY command